tests/test_reverse_fk_relation_crudl_operations_are_supported.py

```
@@ -0,0 +1,260 @@
+    )
+    book: models.Book = models.Book.objects.create(
+        title="Some book",
+        isbn="9783161484100",
+        publication_date="2021-01-01",
+        publisher=publisher,
+    )
+    library: models.Library = models.Library.objects.create(
+        name="Some library",
+        address="Some address",
+    )
+    book_copy: models.BookCopy = models.BookCopy.objects.create(
```

Ruff (0.8.2)

Local variable `book_copy` is assigned to but never used

Remove assignment to unused variable `book_copy`

(F841)